rotkehlchen/db/settings.py

```diff
@@ -330,7 +331,13 @@ def db_settings_from_dict(
             specified_args[key] = [HistoricalPriceOracle.deserialize(oracle) for oracle in oracles]
         elif key == 'non_syncing_exchanges':
             values = json.loads(value)
-            specified_args[key] = [ExchangeLocationID.deserialize(x) for x in values]
+            non_syncing_exchanges = []
+            for exchange_info in values:  # exclude possible leftovers of removed exchanges
+                location_id = ExchangeLocationID.deserialize(exchange_info)
+                if location_id.location in SUPPORTED_EXCHANGES:
```

Comment:
Sorry to be annoying here, but I also meant to delete it from the DB. Since this will stay here forever and always hit this special case